File: even_or_odd.rb
array = 10.times.map{Random.rand(100)}

def even_or_odd(ar)
    even_ar =[]
    odd_ar = []
    ar.each do |num|
        num % 2 == 0 ? even_ar << num : odd_ar << num
    end
    results = []
    results << even_ar
    results << odd_ar
    return results
end

p even_or_odd(array)

File: even_or_odd.js
let array = Array.from({length: 40}, () => Math.floor(Math.random() * 40));

function evenOrOdd(ar) {
  var even = [];
  var odd = [];
  for (n in ar) {
    n % 2 == 0 ? even.push(n) : odd.push(n)
  }
  console.log(even)
  console.log(odd)
}